### DISCUSSION TITLE
Small package with support for PostgreSQL, MySQL and SQLite

### DISCUSSION THREAD

**Discussion by frozenminds** [looplab/eventhorizon#413] [General]
Hi,

In an attempt to learn Go and start building a personal project, I discovered EventHorizon.
First, thanks to all the authors and contributors.

I know there is a small event source [implementation](https://github.com/giautm/eh-pg) (needs updating) for Postgres and this PR #344 which seems abandoned. However, I wanted to use the opportunity to learn and write some Go from scratch without being influenced by the existing code.

So here is my attempt to build an event store and a repo for EventHorizon using [BUN](https://bun.uptrace.dev/). Tests are passing with Postgres, MySQL and SQLite. I didn't bother too much with MSSQL because it's my least priority (probably not too much to fix).

See it here:
- https://pkg.go.dev/github.com/frozenminds/eh-bun
- ~~https://github.com/frozenminds/eh-bun~~ moved to https://codeberg.org/frozenminds/eh-bun

I'm not sure about the code quality, hope it's decent. Any improvements under any form are welcome. If it's good enough at some point, you can also include it in the main repo.

Thanks!

**Comment by frozenminds** [comment]
Well, I guess there's nobody in here. Thread closed!

**Comment by maxekman** [comment]
You are right, I don’t maintain this very actively. The package is however used in various systems, so it’s not abandoned.

I’m happy to link to your implementations in the readme (independently of any quality checks, that can be left to the potential users). If you want that just open a PR and I’ll approve it right away.

I did not take action on this thread as I did not see a question or request. Hoe you understand.

Thanks,
Max
